roles/keycloak/README.md

```diff
@@ -75,20 +76,44 @@ The following variables are _required_ only when `keycloak_db_enabled` is True:
 |`keycloak_db_user` | username for connecting to postgres | `keycloak-user` |
 |`keycloak_db_pass` | password for connecting to postgres | `keycloak-pass` |
 
+The following variable can be used to install Keycloak or Red Hat Single Sign-On from local path:
+| Variable | Description | Example |
+|:---------|:------------|:---------|
+|`zip_file_local_path` | Full local path of upstream(Keycloak) or Red Hat Single Sign-On zip file on Ansible control plane | `tmp/rhsso/rh-sso-7.5-server-dist.zip` |
+
+The following variable can be used to install Red Hat Single Sign-On from source via url, auth support is not added right now.
+| Variable | Description | Example |
+|:---------|:------------|:---------|
+|`rhsso_source_download_url` | URL to download Red Hat Single Sign-On zip file from source | `http://localhost:8081/nexus/rhsso/rh-sso-7.5-server-dist.zip` |
 
```

Comment:
The idea here would be to always rely on a local file for installation; from where, three execution paths open: 
1) download from rhn to that temp file, then distribute the file to nodes
2) download from alternate uri to that temp file, then distribute the file to nodes
3) the file is there and we are offline, just distribute

I'll forward fix before a new release